sync/downstream.py

```diff
@@ -1069,3 +1075,32 @@ def update_pr(git_gecko, git_wpt, sync, action, merge_sha, base_sha, merged_by=N
     except Exception as e:
         sync.error = e
         raise
+
+
+def get_temporary_metadata(sync):
+    pr_id = sync.pr
+    _, statuses = env.gh_wpt.get_combined_status(pr_id)
+    for status in statuses:
+        if status.context == "Taskcluster (pull_request)":
+            taskgroup_id = status.target_url.rsplit('/', 1)[1]
+            taskgroup = TaskGroup(taskgroup_id)
+            task_name = 'wpt-firefox-nightly-results'
+
+            def t_filter(x):
+                return x['task']['metadata']['name'] == task_name
+            tasks = taskgroup.view(filter_fn=t_filter)
+
+            if len(tasks) != 1:
+                logger.error("Could not find the TaskCluster task for %s" % task_name)
+                return False
+            get_wpt_report(tasks, pr_id)
+            try:
+                log_path = tasks.tasks[0]['status']['runs'][0]['_log_paths']['wpt_report.json']
+            except KeyError:
+                logger.warning("Log path not found for downloaded logs from PR Taskcluster run")
+                return False
+            sync.update_metadata([log_path])
```

Comment:
Some thought is required here; for new tests it's probably fine. For other cases we might want to worry about how platform differences are going to be handled. We should at least run the experiment by (by hand) taking a try push for a PR from the sync bot and updating it using the results from upstream and ensuring it's doing what we expect.

Reply:
Should we disable this line for the time being and so that we can check manually till we are confident?

Reply:
That makes sense.